src/20_Components/amp-list.html

```diff
@@ -25,6 +25,38 @@
     bottom: 0;
     right: 0;
   }
+  <!--
+    #### CSS for the amp-list loader
```

Comment:
Yup, saw `The text (CDATA) inside tag 'style amp-custom' contains 'html comments', which is disallowed. (see https://www.ampproject.org/docs/reference/spec.html#stylesheets)` in the Travis logs, will fix.